compiler/stable_mir/src/mir/visit.rs

```diff
@@ -382,6 +390,22 @@ pub trait MirVisitor {
         let _ = args;
     }
 
+    fn super_var_debug_info(&mut self, var_debug_info: &VarDebugInfo) {
```

Comment:
Do you mind de-structuring this variable? We've been trying to follow the compiler visitor convention to keep the visitor up-to-date. I.e., the first line of this function should look like:
```rust
let VarDebugInfo { source_info, composite, value, name: _, argument_index: _ } = var_debug_info;
```